web/packages/teleport/src/config.ts

```diff
@@ -557,8 +557,7 @@ const cfg = {
   },
 
   getUsersRoute() {
-    const clusterId = cfg.proxyCluster;
-    return generatePath(cfg.routes.users, { clusterId });
```

Comment:
Why is `clusterId` being removed from here?

Reply:
the user [route](https://github.com/gravitational/teleport/blob/17b395ecaa0dabd70781084d35e7081f4335d48d/web/packages/teleport/src/config.ts#L161) never needed it 😅 (or perhaps some left over from long ago?)